server/config/environment/development.js

```diff
@@ -17,5 +17,8 @@ module.exports = {
   // For testing verification
   // attendanceVerificationRatio: 1,
 
-  seedDB: false
+  seedDB: false,
+  // Github token for worker, local.env.js kept returning undefined but it worked once I added it here...
+  GITHUB_WORKER_TOKEN: 'YOUR_KEY',
```

Comment:
Check out server/config/environment/index.js for examples of how to read from the environment variables to set a config variable.